microsoft-365/compliance/dlp-conditions-and-exceptions.md

```diff
@@ -140,6 +139,7 @@ This table describes the actions that are available in DLP.
 
 |action in DLP|action parameters in Security & Compliance PowerShell|property type|description|
 |---|---|---|---|
+|Restrcit Access or Encrypt content in Microsoft 365 locations|BlockAccess|First property: *Boolean* <br/><br/> Second property: *BlockAccessScope*|This will allow you to block the access or encrypt the content to the specified users.|
```

Comment:
```suggestion
|Restrict access or encrypt content in Microsoft 365 locations|BlockAccess|First property: *Boolean* <br/><br/> Second property: *BlockAccessScope*|This will allow you to block the access or encrypt the content to the specified users.|
```